Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -213,6 +213,11 @@ public final class XcodeBuildController: XcodeBuildControlling {
             .asSingle()
     }
 
+    public func version() -> Observable<SystemEvent<String>> {
```

Comment:
Because CI environments have Xcode, I think we can add an integration test here that ensures the observable returns a value. We don't need to assert the equality of the value because that'll create a dependency between the test and the environment that we don't want to have.

Reply:
Could you please point me to an example of a similar test? do you mean to add a unit test or a fourier one?

Reply:
```swift
class XcodeBuildControllerIntegrationTest: TuistTestCase {
  func test_version_doesnt_error() throws {
    try subject.version().toBlocking()
  }
}
```